backend/tests/mw_drw_proxy/proxy.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import typing
+
+from flask import Flask, request
+import logging
+
+from diem import identifier
+
+from tests.e2e_tests import UserClient
+from tests.mw_drw_proxy.helpers import (
+    DEFAULT_KYC_INFO,
+    convert_offchain_to_user_info,
+    convert_user_info_to_offchain,
+    error_response,
+    unique_username,
+)
+
+APP_NAME = __name__.split(".")[0]
+LOG = logging.getLogger(APP_NAME)
+logging.basicConfig(level=logging.INFO)
+
+app = Flask(APP_NAME)
+
+STATE: typing.Dict[str, UserClient] = {}
+
+DRW_URL_PREFIX = os.getenv("DRW_URL_PREFIX", "https://demo-wallet.diem.com")
+
+
+@app.route("/reset", methods=["GET", "POST"])
+def reset_test():
+    STATE.clear()
+
+
+@app.route("/accounts", methods=["POST"])
+def create_account():
+    try:
+        data = json.loads(request.data)
+
+        # "{\"type\": \"individual\", \"payload_version\": 1, \"given_name\": \"Tom\", \"surname\": \"Jack\"}"
+        if data.get("kyc_data") is None:
+            kyc_info = DEFAULT_KYC_INFO
+        else:
+            # This may explode, and that's ok
+            try:
+                kyc_info = convert_offchain_to_user_info(json.loads(data["kyc_data"]))
+            except Exception as e:
+                LOG.error(f">>>> ERROR! {e}")
+                raise
+
+        uc = UserClient.create(
+            DRW_URL_PREFIX,
+            unique_username(
+                STATE, "_".join([kyc_info["first_name"], kyc_info["last_name"]]).lower()
+            ),
+            kyc_info=kyc_info,
+            log_fn=LOG.info,
+            randomize=False,
```

Comment:
looks like randomize=True will do same thing did in unique_username.
too many options here looks very confusing how the name is created.
it took me a while to figure out why randomize is False as you wanted to have uniq id appended to the name.